homework.md

```diff
@@ -86,25 +90,25 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+A website where to store and share our code.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Saving changes locally
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual studio
```

Comment:
I think this should be VS Code

Reply:
![image](https://user-images.githubusercontent.com/75181012/101265241-b6d65f80-373c-11eb-9ff9-54c32023430e.png)
Thank you for the feedback, the full name of the software is Visual Studio code (VS Code)
:)